src/block-management/block-picker.js

```diff
@@ -32,6 +33,8 @@ export default class BlockPicker extends Component<PropsType, StateType> {
 	}
 
 	render() {
+		const titleForAdd = 'ADD BLOCK';
+		const titleForReplace = 'REPLACE BLOCK';
```

Comment:
I'm not convinced by this "title" feature, user might pick a block that won't replace the currently selected block, because the block is not replaceable, in that case the block will be added even though it shows "REPLACE BLOCK".

Moreover I think it's really easy to replace a block unintentionally, especially since we don't yet have the `BlockListAppender` component at the end of the list and the (+) button is global to the app currently.

That said, I don't mind much fixing those usability issues later since we're still aiming for alpha.

Reply:
> I'm not convinced by this "title" feature, user might pick a block that won't replace the currently selected block, because the block is not replaceable, in that case the block will be added even though it shows "REPLACE BLOCK".

This wouldn't happen - it should say "REPLACE BLOCK" _only if the block is really going to be replaced_ :) - for that, this check is getting run: https://github.com/wordpress-mobile/gutenberg-mobile/pull/327/files#diff-2d50daf729e068ecebcce113d1ce7f9aR280

So this way there shouldn't be a risk in using the wrong title. On the contrary, leaving the "add block" title for all cases when we're introducing "replacement" as an action for empty editable blocks (`Paragraph`, `Title`, and `Code` for now) seemed misleading as "adding" is not what ends up happening with the introduction of this behavior.

Reply:
Nevermind about replacing a block unintentionally, we check that the block is empty first 👍

Reply:
> This wouldn't happen - it should say "REPLACE BLOCK" only if the block is really going to be replaced :)
Oh you're right, again we check that the block is empty, missed that 🙇